addons/hr/models/hr_employee_public.py

```diff
@@ -60,7 +60,7 @@ class HrEmployeePublic(models.Model):
     is_manager = fields.Boolean(compute='_compute_is_manager')
     is_user = fields.Boolean(compute='_compute_is_user')
 
-    employee_id = fields.Many2one('hr.employee', 'Employee', compute="_compute_employee_id", search="_search_employee_id", compute_sudo=True)
+    employee_id = fields.Many2one('hr.employee', 'Employee', store=True)
```

Comment:
`store=True` is the default, so it's not necessary.
And because the field is part of the SQL view of the model, it should be readonly.
```suggestion
    employee_id = fields.Many2one('hr.employee', 'Employee', readonly=True)
```